js/script.js

```diff
@@ -22,10 +30,87 @@ function closeMenu() {
   closeBtn.style.display = 'none';
   main.style.display = 'block';
   intro.style.display = 'block';
+  footer.style.display = 'block';
   navBarBlock.style.height = 'auto';
 }
 
 closeBtn.addEventListener('click', closeMenu);
 navBarList.forEach((e) => {
   e.addEventListener('click', closeMenu);
 });
+
+/* CREATE SPEAKERS ARRAY OF OBJECTS  */
+
+/* DYNAMIC SPEAKERS SECTION */
+article.id = 'featured-speakers';
+article.innerHTML = `
+  <h2>Featured Speakers</h2>
+  <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+
+  <section class="speaker-section">
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_01.png" alt="speaker 1"></div>
+        <div class="speaker-details">
+          <h3>Yohai Benkler</h3>
+          <h4>Professor, Harvard Law School</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>Focusing on a common approach in a networked environment, we created the concept of co-production based on open source software and sharing, such as Wikipedia.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_02.png" alt="speaker 2"></div>
+        <div class="speaker-details">
+          <h3>Jeon Gilnam</h3>
+          <h4>Professor Emeritus, Korea Advanced Institute of Science and Technology (KAIST)</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>He developed SDN, the first Internet protocol network in Asia, and led Korea's first dedicated line Internet connection in 1990, opening the Internet era in earnest.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_03.png" alt="speaker 3"></div>
+        <div class="speaker-details">
+          <h3>Soyoung Roh</h3>
```

Comment:
- [ ] Kindly make sure you implement a function that iterates through the speakers' list on window load and creates HTML elements with the data from each object as required for the project.

- [ ] Also, the dynamic content should be attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()`.
For a short remember about the ` DYNAMIC SPEAKERS`, you can use the same logic as what you did [while creating your portfolio on the **cards' project**](https://github.com/miltonHenschel/myPortfolio/blob/main/js/projects.js).

Reply:
Yes, thanks @tresorsawasawa.